library.js

```diff
@@ -0,0 +1,128 @@
+class Book {
+  constructor(title, author, pages, read) {
+    this.title = title;
+    this.author = author;
+    this.pages = pages;
+    this.read = read;
+  }
+}
+
+function changeStatus(status) {
+  const index = status.parentElement.parentElement.rowIndex;
+  const readAttribute = JSON.parse(localStorage.books);
+  if (readAttribute[index - 1].read === true) {
+    readAttribute[index - 1].read = false;
+    status.classList.remove('btn-success');
+    status.classList.add('btn-danger');
+    status.innerHTML = 'No';
+  } else {
+    readAttribute[index - 1].read = true;
+    status.classList.remove('btn-danger');
+    status.classList.add('btn-success');
+    status.innerHTML = 'Yes';
+  }
+  localStorage.setItem('books', JSON.stringify(readAttribute));
+}
+
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
+
+class UI {
+  static displayBooks() {
+    const books = Store.getBooks();
+    books.forEach(book => UI.addBookToList(book));
+  }
+
+  static addBookToList(book) {
+    const table = document.querySelector('#book-list');
+    const newRow = table.insertRow(-1);
+    const title = newRow.insertCell(-1);
+    title.innerHTML = book.title;
+    const author = newRow.insertCell(-1);
+    author.innerHTML = book.author;
+    const pages = newRow.insertCell(-1);
+    pages.innerHTML = book.pages;
+    const read = newRow.insertCell(-1);
+    const status = document.createElement('button');
+    if (book.read === true) {
+      status.classList.add('btn', 'btn-success');
+      status.innerHTML = 'Yes';
+    } else {
+      status.classList.add('btn', 'btn-danger');
+      status.innerHTML = 'No';
+    }
+    read.appendChild(status);
+    const deleteBook = newRow.insertCell(-1);
+    const deleteButton = document.createElement('a');
+    deleteButton.innerHTML = 'X';
+    deleteButton.classList.add('btn', 'btn-danger', 'btn-sm', 'delete');
+    deleteBook.appendChild(deleteButton);
+
+    status.addEventListener('click', () => {
+      changeStatus(status);
+    });
+    deleteButton.addEventListener('click', () => {
+      deleteBookFromLocalstorage(deleteButton);
+    });
+  }
```

Comment:
Don't mix the `addBookToLibrary` function with the code that renders the dom. Separate it.

Reply:
The function `addBookToList` it's already separated from the `addBook`